### ISSUE TITLE
[yt-4.0] Update docs TOC to point to yt4differences

### ISSUE THREAD

**Issue by chrishavlin** [yt-project/yt#2734]
https://yt-project.org/docs/dev/ currently shows a link to https://yt-project.org/docs/dev/yt3differences.html, should link to https://yt-project.org/docs/dev/yt4differences.html ?

**Comment by chrishavlin** [comment]
I think the fix is as simple replacing 3 with 4 in 
```
      <tr valign="top">
       <td width="25%">
         <p>
           <a href="yt3differences.html">yt 3.0</a>
         </p>
       </td>
       <td width="75%">
         <p class="linkdescr">How yt-3.0 differs from past versions</p>
       </td>
     </tr>
``` 
within `doc/source/index.rst`

If that's right, I can put in a PR, but I don't have sphinx set up locally to test (yet).

**Comment by chrishavlin** [comment]
Ah, doesn't seem like `yt4differences.html` is finished yet, so won't put in a PR for the TOC fix now.

**Comment by munkm** [comment]
This was resolved in yt-project/yt#3075 with commit 07f2f563d36c6ef5bf8a54c06b2bf86d9bafd12b
